OpenRA.Mods.Common/Widgets/Logic/Ingame/SupportPowerTooltipLogic.cs

```diff
@@ -57,12 +57,23 @@ public SupportPowerTooltipLogic(Widget widget, TooltipContainerWidget tooltipCon
 				name = sp.Info.Description;
 				desc = sp.Info.LongDesc.Replace("\\n", "\n");
 
+				var hotkeyWidth = 0;
 				var hotkey = icon.Hotkey;
-				var hotkeyText = "({0})".F(hotkey.DisplayString());
-				var hotkeyWidth = hotkey.IsValid() ? nameFont.Measure(hotkeyText).X + 2 * nameLabel.Bounds.X : 0;
-				hotkeyLabel.GetText = () => hotkeyText;
-				hotkeyLabel.Bounds.X = nameFont.Measure(name).X + 2 * nameLabel.Bounds.X;
-				hotkeyLabel.Visible = hotkey.IsValid();
+				hotkeyLabel.Visible = false;
+
+				if (hotkey != null)
+				{
+					var key = hotkey.GetValue();
+					var hotkeyText = "({0})".F(key.DisplayString());
+					if (key.IsValid())
+					{
+						hotkeyWidth = nameFont.Measure(hotkeyText).X + 2 * nameLabel.Bounds.X;
+						hotkeyLabel.Visible = true;
+					}
+
+					hotkeyLabel.GetText = () => hotkeyText;
+					hotkeyLabel.Bounds.X = nameFont.Measure(name).X + 2 * nameLabel.Bounds.X;
```

Comment:
var nameWidth = font.Measure(name).X; could be used and reused below like in ProductionTooltipLogic.